crates/iota-sdk-ffi/src/crypto/secp256r1.rs

```diff
@@ -129,6 +125,28 @@ impl Secp256r1VerifyingKey {
     pub fn to_pem(&self) -> Result<String> {
         Ok(self.0.to_pem()?)
     }
+
+    pub fn verify(&self, message: Vec<u8>, signature: &Secp256r1Signature) -> Result<()> {
```

Comment:
I wonder what this return type looks like in python 🤔

Reply:
But raises an error